controller/error_pages.rst

```diff
@@ -9,13 +9,13 @@ In Symfony applications, all errors are treated as exceptions, no matter if they
 are just a 404 Not Found error or a fatal error triggered by throwing some
 exception in your code.
 
-If your app has the `TwigBundle`_ installed, a special controller handles these
+If your app has the `FrameworkBundle`_ installed, a special controller handles these
```

Comment:
I think we can completely remove this paragraph + composer install. Using the framework without FrameworkBundle is impossible